release-notes/opensearch-security.release-notes-2.5.0.0.md

```diff
@@ -0,0 +1,17 @@
+## 2023-01-17 Version 2.5.0.0
+
+Compatible with OpenSearch 2.5.0
+
+### Enhancements
+* [Backport 2.x] When excluding fields also exclude the term + .keyword ([#2377](https://github.com/opensearch-project/security/pull/2377))
```

Comment:
We should remove the [Backport 2.x] for the release notes.

Reply:
Done.